instrumentation/aws-sdk/aws-sdk-1.11/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v1_11/SnsAttributesExtractor.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.awssdk.v1_11;
+
+import com.amazonaws.AmazonWebServiceRequest;
+import com.amazonaws.Request;
+import com.amazonaws.Response;
+import io.opentelemetry.api.common.AttributesBuilder;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.instrumentation.api.instrumenter.AttributesExtractor;
+import io.opentelemetry.semconv.SemanticAttributes;
+import javax.annotation.Nullable;
+
+public class SnsAttributesExtractor implements AttributesExtractor<Request<?>, Response<?>> {
+  @Override
+  public void onStart(AttributesBuilder attributes, Context parentContext, Request<?> request) {
+    String destination = findMessageDestination(request.getOriginalRequest());
+    if (destination != null) {
```

Comment:
Usually we use https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/1ee51e410e90de9a54e1f840c63b222d390f921f/instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/internal/AttributesExtractorUtil.java#L22-L27 which does the null check